spice/spice.py

```diff
@@ -146,28 +146,20 @@ def _validate_model_aliases(model_aliases, clients):
 
 class Spice:
     def __init__(self, default_model=None, default_provider=None, model_aliases=None):
+        """
+        Initializes the Spice client with optional default settings.
+        
+        :param default_model: The default model to use for calls if no model is specified.
+        :param default_provider: The default provider to use if no provider is specified.
+        :param model_aliases: A dictionary mapping model aliases to their respective model names and providers.
+        """
         self._default_model = default_model
+        self._default_provider = default_provider
+        self._model_aliases = model_aliases
+        self._client_manager = ClientManager()
```

Comment:
how about we just have Spice only set _client_manager, not the other 3? just leave those internal to ClientManager